_posts/2020-06-15-3D-Printing-During-Times-Of-Corona.md

```diff
@@ -0,0 +1,275 @@
+</div>
+<br/>
+
+This mask fits very well and although it requires quite a bit of extra work I prefer it over the first version!
+It sits very comfortable and does not require printing with TPU.
+The silicone molding process does take some trial an error but the mask shown in the pictures is from the first attempt!
+
+<div style="text-align: center; margin: 0px auto;">
+     <a href="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0584.jpg' | prepend: site.baseurl }}" data-lightbox="fdm" data-title="Printed mask & Silicone mold">
+        <img alt="stack" src="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0584.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; width: 20.5%; display: inline-block;">
+    </a>
+    <a href="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0486.jpg' | prepend: site.baseurl }}" data-lightbox="fdm" data-title="Mask with silicone seal 1">
+        <img alt="stack" src="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0486.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; width: 11.5%; display: inline-block;">
+    </a>
+    <a href="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0487.jpg' | prepend: site.baseurl }}" data-lightbox="fdm" data-title="Mask with silicone seal 2">
+        <img alt="stack" src="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0487.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; width: 11.5%; display: inline-block;">
+    </a>
+     <a href="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0488.jpg' | prepend: site.baseurl }}" data-lightbox="fdm" data-title="Mask with silicone seal 3">
+        <img alt="stack" src="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0488.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; width: 11.5%; display: inline-block;">
+    </a>
+    <a href="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0582.jpg' | prepend: site.baseurl }}" data-lightbox="fdm" data-title="Mask, filter and filter cap">
+        <img alt="stack" src="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0582.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; width: 20.5%; display: inline-block;">
+    </a>
+     <a href="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0583.jpg' | prepend: site.baseurl }}" data-lightbox="fdm" data-title="Assembled mask">
+        <img alt="stack" src="{{ '/img/2020-06-15-3D-Printing-during-corona/IMG_0583.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; width: 11.5%; display: inline-block;">
+    </a>
+</div>
+<br/>
+
+There are many many more mask designs online.
+You can always design a custom mask or look on thingiverse or any other 3D modelling site for designs from other people, the community is very large and very engaged when it comes to prototyping.
```

Comment:
Suggestion: Link to thingiverse
from other people. The community is very large ..